src/services/auth.ts

```diff
@@ -107,6 +107,9 @@ export enum AuthGroup {
   Staff = '360-Staff-SG',
   Student = '360-Student-SG',
   AcademicInfoView = '360-AcademicInfoView-SG',
+  LostAndFoundAdmin = '360-LostAndFoundAdmin-SG',
+  LostAndFoundKiosk = '360-LostAndFoundKiosk-SG',
+  LostAndFoundDevelopers = '360-LostAndFound-Developers-SG',
```

Comment:
Evan added a lost and found developers security group, so we can use this one for now, pending our conversation with him.  I tested it and it works for me 👍